src/js/modules/accounts/reducer.js

```diff
@@ -67,23 +75,14 @@ const reducer = (state = { ...initialState }, action) => {
       return {
         ...state,
         status: {
-          errors: action.payload.response.data.errors,
+          errors: (action.payload.response &&
```

Comment:
Feels like you're handling more than one situation in this action. Consider splitting it up?

Reply:
The two cases are based on whether API or client-app screwed up. Both cases call for a `REJECTED` action.

Reply:
Could you put a comment explaining it? I'm still not clear on what is happening